app/src/main/res/values-fr/strings.xml

```diff
@@ -37,12 +37,12 @@
     <string name="chooseTheme">Thème</string>
     <string name="website">Site Web</string>
     <string name="website_desc">En savoir plus sur Exodus Privacy</string>
-    <string name="srcCode_desc">Signaler des bugs, contribuer au code source</string>
+    <string name="srcCode_desc">Signaler des bugs, contribuer ou consulter le code source</string>
```

Comment:
Sur l'application sa prend 2 lignes, peut être réécrire le texte en anglais pour le simplifier.
Par exemple:
- View the source code on Github
- Fork the project on Github
- Report a bug

Reply:
Les descriptions en anglais et français ont été simplifiées